Rakefile

```diff
@@ -91,7 +91,7 @@ end
 
 # Bundle the freetds sources to avoid download while gem install
 task gem_build_path(gemspec) do
-  add_file_to_gem(gemspec, "ports/archives/freetds-0.91.112.tar.gz")
+  add_file_to_gem(gemspec, "ports/archives/freetds-0.95.64.tar.bz2")
 end
```

Comment:
Sorry, I don't understand your question. This is to include the standard freetds tar file into the gem, so that a simple `gem install` without any further settings can build freetds without an additional download. Any other version will be downloaded automatically, but this particular version is bundled and the download is skipped.